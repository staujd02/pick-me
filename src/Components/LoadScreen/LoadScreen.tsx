import React from 'react';
import logo from '../../Assets/logos/kaleidoscope-2.svg';

const LoadScreen: React.FC = () => 
    <header className="App-header">
        <img src={logo} className="App-logo" alt="logo" />
        <p>Starting Up The Streams...</p>
    </header>

export default LoadScreen;