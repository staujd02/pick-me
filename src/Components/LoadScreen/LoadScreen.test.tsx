import React from 'react';
import LoadScreen from './LoadScreen';
import { ShallowWrapper, shallow } from 'enzyme';
import { formatHTML } from '../../TestUtilities/htmlFormatter';

describe('The Load Screen', () => {
    
    let wrapper: ShallowWrapper;

    beforeEach(() => {
        wrapper = shallow(<LoadScreen />)
    });

    it('renders correctly', () => expect(formatHTML(wrapper.html())).toMatchSnapshot());
});