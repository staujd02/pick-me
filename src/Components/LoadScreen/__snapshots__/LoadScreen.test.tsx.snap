// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`The Load Screen renders correctly 1`] = `
"
<header class=\\"App-header\\">
  <img src=\\"kaleidoscope-2.svg\\" class=\\"App-logo\\" alt=\\"logo\\">
  <p>Starting Up The Streams...</p>
</header>
"
`;
